crates/sui-graphql-rpc/src/types/big_decimal.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::str::FromStr;
+
+use async_graphql::*;
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub(crate) struct BigDecimal(String);
```

Comment:
As per the comment later on -- we shouldn't need this scalar type.

Reply:
That's good to know!